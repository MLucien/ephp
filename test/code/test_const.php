<?php

const DATA = "hello world\n";
print DATA;

define(DATA2, "hello again!\n");
print DATA2;

define("DATA3", "good-bye!\n");
print DATA3;
