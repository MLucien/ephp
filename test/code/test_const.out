hello world

Notice: Use of undefined constant DATA2 - assumed 'DATA2' in {{CWD}}/test/code/test_const.php on line 6
hello again!
good-bye!
